src/components/Shared/AmountInput/AmountInput.tsx

```diff
@@ -1,35 +1,40 @@
-import { ChangeEventHandler, FC, useContext } from "react";
+import { useContext } from "react";
+import type { FC } from "react";
+import type { FieldError, UseFormRegisterReturn } from "react-hook-form";
 import { useTranslation } from "react-i18next";
 import { ReactComponent as SwitchIcon } from "../../../assets/switch-vertical.svg";
 import { AppContext } from "../../../store/app-context";
-import styles from "./AmountInput.module.css";
```

Comment:
Killing these because I think all inputs should have the same style unless it's a very special component. In this case it seemed avoidable.